src/routes/AppRoutes.tsx

```diff
@@ -0,0 +1,24 @@
+import { Route, Routes } from 'react-router'
```

Comment:
```suggestion
import { Route, Routes } from 'react-router-dom'
```

react-router 참조해도 동작은 할텐데, 공식문서를 보면 react-router-dom를 참조하도록 가이드하고 있는거 같아요.